### ISSUE TITLE
Last updated date inconsistent

### ISSUE THREAD

**Issue by gordjw** [magda-io/magda#1979]
### Problem description
Last updated dates on search results page and dataset page are inconsistent.
Are they meant to be the same?

### Problem reproduction steps
**Search (first result):**
https://search.data.gov.au/search?q=ASIC%20%E2%80%93%20Credit%20Representative%20Dataset
Last Updated: 03/12/2018

**Click through to first result**
https://search.data.gov.au/dataset/ds-dga-a3b6d445-3a00-409a-8b49-53dc62b823bf/details?q=ASIC%20%E2%80%93%20Credit%20Representative%20Dataset
Updated: 02/01/2019

### Screenshot / Design / File reference
Search result
![screen shot 2019-01-10 at 1 29 33 pm](https://user-images.githubusercontent.com/1028531/50942174-c959a700-14db-11e9-88a4-818e371fbf12.png)

Dataset
![screen shot 2019-01-10 at 1 29 21 pm](https://user-images.githubusercontent.com/1028531/50942177-cb236a80-14db-11e9-8d4f-936b0b4c0961.png)

**Comment by AlexGilleran** [comment]
Yeah it should be consistent, the indexer is stuck on search.data.gov.au right now because of https://github.com/magda-io/magda/issues/1918. That's been fixed so it should un-stick and catch back up once we've deployed 0.0.51, which is blocked on #1975
